src/lib/logMsg/logMsg.h

```diff
@@ -546,6 +546,27 @@ do                                                                     \
 #endif
 
 
+#ifdef LM_NO_TMP
+#define LM_TMP(s)
+#else
+/* ****************************************************************************
+*
+* LM_TMP - log message
```

Comment:
I miss some comment about the usage of this, eg. "This macro is only to be used in development branches as we work. However must'n *ever* appear in final code. In order words, it must be cleaned before creating pull requests for review".

Or any other wording that captures the same idea.